test/libsolidity/semanticTests/viaYul/detect_mul_overflow_signed.sol

```diff
@@ -61,3 +67,18 @@ contract C {
 // g(int8,int8): -64, -2 -> FAILURE, hex"4e487b71", 0x11
 // g(int8,int8): -2, -63 -> 126
 // g(int8,int8): -2, -64 -> FAILURE, hex"4e487b71", 0x11
+// h(int160,int160): -1, 1 -> -1
+// h(int160,int160): 1, -1 -> -1
+// h(int160,int160): -1, 2 -> -2
+// h(int160,int160): 2, -1 -> -2
+// h(int160,int160): -1, -730750818665451459101842416358141509827966271488 -> FAILURE, hex"4e487b71", 0x11
```

Comment:
When long numbers are involved, a hexadecimal notation would make the test case easier to follow.

Reply:
Yes. I used hexadecimals for the minimum value (`0x80....0`), but when running isoltest, they get either converted to decimal (in the case they are the returned value) or generate a `FAILURE` message which doesn't correspond to what would be expected. I was investigating this and searching for issues/PR that could mention such problem or something similar. So far, I have found that the problem doesn't happen for int256, but happens to other sizes as int8 and int160.

**Examples of outputs of isoltest**

It works for int256: 
```
// f(int256,int256): 0x8000000000000000000000000000000000000000000000000000000000000000, 1 -> 0x8000000000000000000000000000000000000000000000000000000000000000
// f(int256,int256): 0x8000000000000000000000000000000000000000000000000000000000000000, -1 -> FAILURE, hex"4e487b71", 0x11
```
For other sizes, the FAILURE we get does not seem to be because of a detected overflow: 
```
// h(int160,int160): 0x8000000000000000000000000000000000000000, 1 -> FAILURE
  Warning: The call to "h(int160,int160)" returned 

// h(int160,int160): -1, 0x8000000000000000000000000000000000000000 -> FAILURE
  Warning: The call to "h(int160,int160)" returned 

// h(int160,int160): 0x8000000000000000000000000000000000000000, -1 -> FAILURE
  Warning: The call to "h(int160,int160)" returned 
 
// g(int8,int8): 1, 0x80 -> FAILURE
  Warning: The call to "g(int8,int8)" returned 

// g(int8,int8): -1, 0x80 -> FAILURE
  Warning: The call to "g(int8,int8)" returned
```
Also, isoltest converts to decimal when `0x80...0` is the expected return value:
```
// h(int160,int160): 0x4000000000000000000000000000000000000000, -2 -> -730750818665451459101842416358141509827966271488
  Warning: Type or size of parameter(s) does not match.
  Warning: The call to "h(int160,int160)" returned 
  [ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,80,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
// h(int160,int160): -2, 0x4000000000000000000000000000000000000000 -> -730750818665451459101842416358141509827966271488
  Warning: Type or size of parameter(s) does not match.
  Warning: The call to "h(int160,int160)" returned 
  [ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,ff,80,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]

```